src/Simplex/Chat.hs

```diff
@@ -2127,6 +2130,7 @@ chatCommandP =
     <|> "/_accept " *> (APIAcceptContact <$> A.decimal)
     <|> "/_reject " *> (APIRejectContact <$> A.decimal)
     <|> "/_call invite @" *> (APISendCallInvitation <$> A.decimal <* A.space <*> jsonP)
+    <|> ("/call @" <|> "/call ") *> (SendCallInvitation <$> displayName <*> pure defaultCallType)
```

Comment:
Parse option to call with audio?

Reply:
maybe later, it's for testing, UX is pretty horrible